.gitignore

```diff
@@ -20,7 +20,17 @@
 node_modules/
 npm-debug.*
 dist/
+
+# Yarn (uncomment for zero-install: https://stackoverflow.com/a/58550620)
 .yarn/
+#!.yarn/cache
+#!.yarn/patches
+#!.yarn/plugins
+#!.yarn/releases
```

Comment:
You should #! this too so 3.3.1 is cloned everywhere

Reply:
New guys need this because you can't install for the first time withou .yarn/versions/yarn-3.3.1.cjs